src/components/grid-aware/TwoParagraphBlock/TwoParagraphBlock.module.css

```
@@ -0,0 +1,143 @@
+/* Background bleed */
+
+.bleedWrapper {
+  display: grid;
+  grid-template-columns: 1fr minmax(var(--grid-content-min-width), var(--grid-content-max-width)) 1fr;
+  grid-template-rows: auto 0;
+  margin: auto;
+  max-width: var(--grid-bleed-max-width);
+}
+
+.bleedBackground {
+  background: var(--color-gray-300);
+  grid-column: 1 / span 2;
+  grid-row: 1 / -1;
+}
+
+.bleedMainContent {
+  grid-column: 2;
+  grid-row: 1;
+  min-width: 0;
+}
+
+@media (--tablet-and-down) {
+  .bleedWrapper {
+    grid-template-columns: 40px 1fr;
+  }
+
+  .bleedBackground {
+    grid-column: 1 / span 3;
```

There are two columns specified above at line 25. Spanning by 3 in column one is too much.

```suggestion
    grid-column: 1 / span 2;
```